docker-compose.yaml

```diff
@@ -67,7 +67,7 @@ services:
       - ${LOCAL_ROOT}:${LOCAL_ROOT}
       - data:${CONFIG_ROOT}
   server:
-    image: airbyte/server:dev
+    image: airbyte/server:${VERSION}
```

Comment:
@subodh1810 this was an accidental merge from the automatic migration PR yes?

Reply:
Yeah sorry about that! will raise a PR to fix this